addons/crm_mail_plugin/controllers/crm_client.py

```diff
@@ -1,64 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import http
-from odoo.http import request
-from odoo.tools import html2plaintext
-
-from .mail_plugin import MailPluginController
-
-
-class CrmClient(MailPluginController):
-
-    @http.route(route='/mail_client_extension/log_single_mail_content',
```

Comment:
If someone keeps the old addin version but updates their odoo, what happens ? Shouldn't we keep a legacy.py file holding redirections to the new routes that are anyway generic ?